public/app/plugins/panel/graph/graph.ts

```diff
@@ -158,7 +158,7 @@ function graphDirective(timeSrv, popoverSrv, contextSrv) {
 
       function processRangeHook(plot) {
         var yaxis = plot.getYAxes();
-        if (yaxis.length > 1 && panel.yaxes[1].align !== null) {
+        if (yaxis.length > 1 && 'align' in panel.yaxes[1] && panel.yaxes[1].align !== null) {
```

Comment:
Should be able to change this to:
```javascript
if (yaxis.length > 1 && yaxis[1].align && yaxis[1].align !== null) {
```

Reply:
'align' not transferred from panel.yaxis to plot.yaxis. But I will correct the presence of the attribute to simple read.

Reply:
Oh didn't spot that. Great